.circleci/config.yml

```diff
@@ -29,7 +29,7 @@ workflows:
             branches:
               only: /(^renovate-.*|^nori/.*)/
             tags:
-              only: /^v\d+\.\d+\.\d+(-.+)?/
+              ignore: /.*/
```

Comment:
**nitpick**: we can remove the `tags` stanza completely as all tags are ignored by default iiuc